### ISSUE TITLE
Unable to import Custom Python service module

### ISSUE THREAD

**Issue by kdhansen** [ajshort/vscode-ros#16]
Sorry, if this is obvious to seasoned users. I am just trying out VSCode with my ROS development.

I have defined a .msg and .srv in my package. I have built the workspace, which works fine in the created terminal. But running a python debugging session stops at the import of the custom service module. I have tried the "ROS: Update Python Path" command, but that does not seem to include the built module. If I create a terminal with "ROS: Create Terminal" and "> source devel/setup.bash" in that terminal, the node runs fine.

**Comment by beetleskin** [comment]
Yes, the plugin lacks to set the _PYTHONPATH_ variable .. it only sets "python.autoComplete.extraPaths" in the settings.json.
